static/_redirects

```diff
@@ -189,7 +189,8 @@
 /docs/reference/generated/kubernetes-api/v1.15/    https://v1-15.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/  301
 /docs/reference/generated/kubernetes-api/v1.16/    https://v1-16.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/  301
 /docs/reference/generated/kubernetes-api/v1.17/    https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/  301
-
+# "latest" API version is a transparent redirect (i.e. rewrite) to the current version.
+/docs/reference/generated/kubernetes-api/latest/*    /docs/reference/generated/kubernetes-api/v1.18/:splat  200
```

Comment:
The only problem as I see it ... we will need to revise this hard coded version number on every release, because `latest` is a relative term.

Reply:
That's true. I'm happy to automate it via the `copyapi` target in the Makefile in reference-docs, but I figured that somebody would be in this file adding an entry just above this. For example, when 1.19 comes out, someone will already be adding a 1.18 link right there, and could easily update the latest at the same time.

Reply:
Here's a way to keep it updated automatically. https://github.com/kubernetes-sigs/reference-docs/pull/160

Reply:
Hello @joelsmith .
I will read through the changes.
I am uncertain about using `latest` as well as adding onto the `copyapi` target to edit the `_redirects` file.
What is the advantage of providing a redirect from `latest`?
I realize that there will be updates per release to make sure that the links (that users previously depended on) are correct.

Reply:
Perhaps the user story would be a good place to start:

As a user, I would like to be able to create links to Kubernetes API object docs, and have those links continue to work for as long a time as possible. I am frustrated when I create a link, and then 15 months later, it stops working because the version of the API that I linked to is no longer supported. I would like my link to go to the current version of the API object I am interested in. If I had created a link to the latest version of the pod spec in the API docs on March 20, 2020 (15 months ago), that link would today be broken, since the latest version at the time was 1.13, which is no longer supported (and no longer hosted or redirected).  (See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#pod-v1-core for an example of such a broken link).

So my proposed PR (with or without the makefile target to auto-update latest) is just one way of supporting that user story. I'm sure there are lots of other good ways.  This PR on its own will provide a (relatively) stable link target. Of course, if the API reference link scheme changes, then external links will all be broken, but this change hopefully increases the likelihood that a 5-year-old link might actually still work 5 years from now.  And who knows? Maybe we'll be able to accommodate any such link schemes on a best-effort basis via redirects just as we already do for moved content.

The other PR in the makefile target would auto-update the redirect, but is not a requirement for this PR (IMO). The new version process could include a step to manually update it, just as it must already include a step to manually add a redirect rule for the just-prior version.

Since nothing links to the "latest" versions of the API docs in this change, it's only useful to people who know about it, but if it turns out to be a good docs feature, we can tell people about it by providing links to it that people can start to use.